ci/pipelines/pr.yml

```diff
@@ -1387,11 +1386,12 @@ jobs:
         outputs:
           - name: pacts
         run:
-          path: sh
+          path: bash
           dir: src
           args:
             - -ec
             - |
+              # shellcheck disable=SC1091
```

Comment:
do we actually need this comment?

Reply:
Yep, this comment is what skips the shellcheck failure. Unless we can fix it instead?

Reply:
Nah, it's ok. I'll approve the PR.